server/src/configs/env.js

```diff
@@ -7,4 +7,4 @@ const configs = {
   clientUrl: process.env.CLIENT_URL || 'http://localhost:3000',
```

Comment:
wouldn't it be better if we directly connect the 'tripen-dev.netlify.com' here?

Reply:
we will store that URL in .env file. instead of assigning the url itself here.